addons/mail/static/src/models/user_setting.js

```diff
@@ -150,7 +150,7 @@ registerModel({
          */
         async setThresholdValue(voiceActivationThreshold) {
             this.update({ voiceActivationThreshold });
-            this.env.services.local_storage.setItem('mail_user_setting_voice_threshold', voiceActivationThreshold);
+            this.messaging.browser.localStorage.setItem('mail_user_setting_voice_threshold', voiceActivationThreshold);
```

Comment:
Docstring says `voiceActivationThreshold` is `float`, and `legacy_local_storage` service applied `JSON.stringify(value)`.

The `window.localStorage.setItem(keyName, keyValue)` needs `keyValue` as string, so it's important to type-cast here.